### DISCUSSION TITLE
3.2 tft display not working at all

### DISCUSSION THREAD

**Discussion by vijaysam13** [Bodmer/TFT_eSPI#1208] [Compatible displays and setup files]
I recently purchased a 3.2" tft lcd display with ili9341 driver and 16 bit parallel interface.
I am not able to make it work with Uno or ESP32
Appreciate Support or if any one can share working library and connection details..
Pin out detais are below:
![Screenshot_20210524-063750_Chrome](https://user-images.githubusercontent.com/84695970/119294681-b46db080-bc5d-11eb-84fb-245f0db7573f.jpg)

**Comment by Bodmer** [comment]
You have a 16 bit parallel interface display which is not supported with full functionality.

There is a pseudo 16 bit mode that can be used. See [this thread](https://github.com/Bodmer/TFT_eSPI/issues/299).
